### PR TITLE
Update billing portal descriptions

### PR DESCRIPTION
**Pull request by noon-dawg**
## Summary

This pull request reworks the billing portal descriptions, and related files, to conform to the [writing style](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md).

## Links

N/A

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

**Review by ashkarpetin — DISMISSED** [review]

File: openapi/components/schemas/Common/CommonBillingPortal.yaml [hunk 1]
```diff
@@ -4,94 +4,108 @@ required:
...
+      make it human-readable. Example: `https://company.com/billing-portal/{slug}`.
     type: string
     minLength: 5
     maxLength: 100
   websiteId:
-    description: Billing portal website ID.
+    description: |-
+      ID of the website that is associated with the billing portal. A
+      website is where your organization obtains a customer. It is
+      also the processor account and billing descriptor used for payment
```
Comment (adamaltman) [inline openapi/components/schemas/Common/CommonBillingPortal.yaml:22]:
```suggestion
      also the gateway account and billing descriptor used for payment
```
Is this what ou mean?

File: openapi/components/schemas/Common/CommonBillingPortal.yaml [hunk 2]
```diff
@@ -4,94 +4,108 @@ required:
...
     readOnly: true
     allOf:
       - $ref: ../ResourceId.yaml
   slug:
-    description: >-
-      The url path segment that will be appended to the billing portal
-      url (e.g. `https://rebilly.com/billing-portal/{slug}`) to make it human-readable and simple.
+    description: |-
+      Path segment that is appended to the billing portal URL to help
+      make it human-readable. Example: `https://company.com/billing-portal/{slug}`.
```
Comment (adamaltman) [inline openapi/components/schemas/Common/CommonBillingPortal.yaml:14]:
```suggestion
      make it human-readable. Example: `https://example.com/billing-portal/{slug}`.
```
We should not use a real URL in the example unless we have permission.

**Review by adamaltman — CHANGES_REQUESTED** [review]

File: openapi/paths/billing-portals.yaml [hunk 3]
```diff
@@ -6,8 +6,8 @@ get:
   summary: Retrieve billing portals
   operationId: GetBillingPortalCollection
   x-sdk-operation-name: getAll
-  description: |
-    Retrieve a list of billing portals.
+  description: |-
+    Retrieves a list of billing portals.
```
Comment (adamaltman) [inline openapi/paths/billing-portals.yaml:10]:
```suggestion
  description: Retrieves a list of billing portals.
```
In this case of a one-liner, the block syntax isn't needed.

File: openapi/paths/billing-portals.yaml [hunk 4]
```diff
@@ -42,11 +42,11 @@ post:
   summary: Create a billing portal
   operationId: PostBillingPortal
   x-sdk-operation-name: create
-  description: |
-    Create a billing portal.
+  description: |-
+    Creates a billing portal.
```
Comment (adamaltman) [inline openapi/paths/billing-portals.yaml:46]:
```suggestion
  description: Creates a billing portal.
```

File: openapi/paths/billing-portals@{id}.yaml [hunk 5]
```diff
@@ -8,11 +8,11 @@ get:
   summary: Retrieve a billing portal
   operationId: GetBillingPortal
   x-sdk-operation-name: get
-  description: |
-    Retrieve a billing portal with specified identifier string.
+  description: |-
+    Retrieves a billing portal with a specified identifier string.
```
Comment (adamaltman) [inline openapi/paths/billing-portals@{id}.yaml:12]:
```suggestion
  description: Retrieves a billing portal with a specified identifier string.
```

File: openapi/paths/billing-portals@{id}.yaml [hunk 6]
```diff
@@ -31,17 +31,18 @@ put:
   summary: Upsert a billing portal
   operationId: PutBillingPortal
   x-sdk-operation-name: update
-  description: |
-    Create or update (upsert) a billing portal with specified identifier string.
+  description: |-
+    Creates or updates (upserts) a billing portal with a specified
+    identifier string.
```
Comment (adamaltman) [inline openapi/paths/billing-portals@{id}.yaml:36]:
```suggestion
  description: Creates or updates (upserts) a billing portal with a specified identifier string.
```

File: openapi/paths/billing-portals@{id}.yaml [hunk 7]
```diff
@@ -70,11 +71,11 @@ delete:
   summary: Delete a billing portal
   operationId: DeleteBillingPortal
   x-sdk-operation-name: delete
-  description: |
-    Delete a billing portal with specified identifier string.
+  description: |-
+    Deletes a billing portal with a specified identifier string.
```
Comment (adamaltman) [inline openapi/paths/billing-portals@{id}.yaml:75]:
```suggestion
  description: Deletes a billing portal with a specified identifier string.
```

**Review by Boydbueno — APPROVED** [review]

**Review by n-fasano — APPROVED** [review]

**Review by adamaltman — APPROVED** [review]

**Pull request merged**